blazor/autocomplete/filtering.md

```diff
@@ -210,4 +210,73 @@ The AutoComplete component filter queries can be customized. You can also use yo
         await autoObj.FilterAsync(Country, query);
     }
 }
-```
\ No newline at end of file
+```
+
+## Properties
+
+### FilterType
+
+Determines on which filter type, the component needs to be considered on search action.
+
+Default value of [FilterType](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfAutoComplete-2.html#Syncfusion_Blazor_DropDowns_SfAutoComplete_2_FilterType) is `Contains`,  all the suggestion items which contain typed characters to listed in the suggestion popup. 
```

Comment:
Default value of [FilterType](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfAutoComplete-2.html#Syncfusion_Blazor_DropDowns_SfAutoComplete_2_FilterType) is Contains, all the suggestion items which contain typed characters are listed in the suggestion popup.